sdk/core/core/test/cmocka/test_az_credential_client_secret.c

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "az_test_definitions.h"
+#include <az_credentials.h>
+#include <az_credentials_internal.h>
+#include <az_http.h>
+#include <az_http_internal.h>
+#include <az_span.h>
+
+#include <stddef.h>
+
+#include <setjmp.h>
+#include <stdarg.h>
+
+#include <cmocka.h>
+
+#include <_az_cfg.h>
+
+static void test_credential_client_secret(void** state)
+{
+  (void)state;
+
+  az_credential_client_secret credential = { 0 };
+  // init credential_credentials struc
```

Comment:
Apply the fix for this comment again?

Reply:
Ha, good catch, thank you. I wonder how it got here, given that I don't like to comment self-documenting code. I think I got it via some set of either undos or merges, but I think it proves that my eyes are rather trained to ignore comments in code :)

Reply:
Fixed